shared-libs/outbound/test/outbound.spec.js

```diff
@@ -340,10 +341,7 @@ describe('outbound shared library', () => {
           assert.equal(request.post.callCount, 1);
           assert.equal(request.post.args[0][0].url, 'http://test/foo');
           assert.deepEqual(request.post.args[0][0].body, {some: 'data'});
-          assert.equal(request.post.args[0][0].json, true);
-          assert.deepEqual(request.post.args[0][0].headers, {
```

Comment:
I generally prefer to keep test assertions with as much parameter coverage as possible, So here, instead of reducing coverage, can you please update the test to assert other potential headers too instead of reducing the assertion to just one header?